test/core/currency/wallet/currency-wallet.test.ts

```diff
@@ -210,28 +211,25 @@ describe('currency wallets', function () {
 
   it('exposes custom tokens', async function () {
     const log = makeAssertLog()
-    const { config, wallet } = await makeFakeCurrencyWallet()
+    const { config } = await makeFakeCurrencyWallet()
 
     config.watch('customTokens', () => log('customTokens changed'))
-    await wallet.addCustomToken({
+    expect(config.customTokens).deep.equals({})
+
+    const customToken: EdgeToken = {
       currencyCode: 'CUSTOM',
-      currencyName: 'Custom Token',
-      contractAddress:
-        '0X7CD5885327FD60E825D67D32F9D22B018227A208AA3C4819DA15B36B5D5869D3',
-      multiplier: '1000'
-    })
-    log.assert('customTokens changed')
+      displayName: 'Custom Token',
+      denominations: [{ multiplier: '1000', name: 'CUSTOM' }],
+      networkLocation: {
+        contractAddress:
+          '0X7CD5885327FD60E825D67D32F9D22B018227A208AA3C4819DA15B36B5D5869D3'
+      }
+    }
 
+    await config.addCustomToken(customToken)
```

Comment:
The customToken parameter here looks different from before. How is the networkLocation able to change without implementation change; are both network locations values cleaned to the same object in the assertion?

Reply:
No, these are different functions. The `EdgeCurrencyConfig` version takes an `EdgeToken`, but the `EdgeCurrencyWallet` one takes an `EdgeMetaToken`, so the format has to change.